.goreleaser.yaml

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env bash
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+env:
+  # Set env var TAG_OVERRIDE to override the container images' tag.
+  # By default we use Version as the tag.
+  - TAG={{ if index .Env "TAG_OVERRIDE"  }}{{ .Env.TAG_OVERRIDE }}{{ else }}'{{ .Version }}'{{ end }}
+
+before:
+  hooks:
+    - go mod tidy
+
+builds:
+  -
+    id: cert-rotation
+    main: ./cmd/cert-rotation
+    binary: cert-rotation
+    env:
```

Comment:
I think you can set this `env` globally above instead of in each build step.

Reply:
Done